packages/components/src/particles/Banner/index.tsx

```diff
@@ -30,13 +31,19 @@ export default function Banner(props: BannerProps) {
     }
   };
 
-  const bannerClasses = () => {
+  const bannerClasses = (includeBorder: boolean) => {
```

Comment:
Just for curiosity, why not just use `props.withBorder`?

Reply:
there is absolutely no reason why! i just tend to like doing things this way because in my head it makes em easier to test this function in isolation if we were ever that psychotic

Reply:
since it's not being tested int hat way and likely never will lemme just check the props and remove some complesxity